core-bundle/src/Controller/FrontendModule/TwoFactorController.php

```diff
@@ -41,6 +42,10 @@ class TwoFactorController extends AbstractFrontendModuleController
 
     public function __invoke(Request $request, ModuleModel $model, string $section, array $classes = null, PageModel $page = null): Response
     {
+        if (!$this->get('security.helper')->isGranted('IS_AUTHENTICATED_FULLY')) {
+            throw new InsufficientAuthenticationException('User is not fully authenticated');
+        }
```

Comment:
Wouldn't this mean the module generates an exception if you open the page with a REMEMBERME cookie?

Reply:
Unfortunately yes. And since we have no "re-enter your password" screen yet, the user will be redirected to the login module, which will happily tell them that they are already logged in. 😢